data.sql

```diff
@@ -33,4 +33,49 @@ INSERT INTO animals (animals_id, name, date_of_birth, escape_attempts, neutered,
  	5,
  	true,
  	11
+);
+
+INSERT INTO animals (animals_id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES(
+ 	5,
+	'Charmander',
+ 	'2020-02-08',
+ 	0,
+ 	false,
+ 	-11
+);
+
```

Comment:
- [ ] According to the requirements:

![image](https://user-images.githubusercontent.com/57421823/154272406-761f9a06-a531-409a-b5ed-273987770686.png)

It seems like "Blossom" is missing.  Please, kindly add it. 💪